wallet/types/src/keys.rs

```diff
@@ -97,7 +97,7 @@ impl KeychainUsageState {
     /// Increments the last index used in the blockchain until up_to_last_used. This has no effect
     /// if the up_to_last_used is smaller than the self value
     pub fn increment_up_to_last_used(&mut self, up_to_last_used: ChildNumber) {
-        if self.last_used.is_none() || self.last_used.unwrap() < up_to_last_used {
+        if self.last_used.is_none() || self.last_used.expect("must be present") < up_to_last_used {
```

Comment:
I fixed this correctly in Alfie's PR... expect() is not really the right way to do this. Maybe we should pull it from there and put it in a PR of its own... now we have multiple PRs getting in trouble because of this.